docs/source/models/cause.rst

```diff
@@ -298,9 +285,24 @@ scenarios:
   location
 - A simulant enters the simulation by *aging* into the simulation
 
-Prevalence is **not** used to initialize cause model states when a simulant 
-is *born* into a simulation. See the below section on birth prevalence for 
-how cause model states are initialized in this scenario.
+	Prevalence is **not** used to initialize cause model states when a 
+	simulant is *born* into a simulation. See the below section on birth 
+	prevalence for how cause model states are initialized in this scenario.
+
+While GBD results are estimates of *point* prevalence at the year midpoint, 
+cause model initialization occurs continuously throughtout a full simulation 
```

Comment:
Cause model initialization does indeed occur every time step, but prevalence is only used at simulation start.  Afterwards birth prevalence is used.  This might be different if we included something like migration.

Reply:
Updated in last commit!